addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh

```diff
@@ -119,7 +119,8 @@ PIP_TO_INSTALL="
     PyKCS11 \
     vcgencmd \
     RPi.GPIO \
-    rjsmin==1.1.0"
+    rjsmin==1.1.0 \
+    schedule==1.2.1"
```

Comment:
```suggestion
    schedule==1.2.1" \
```
Next this avoid to have 2 line diff

Reply:
what do you mean @qle-odoo  ? I don't see what it changes in term of line diffs :thinking:

Reply:
I think @qle-odoo means that you need to add the `\` already now so that the next time you want to add a package you will have 1 line diff instead of 2 lines diff.

But this won't work because it's one big string so you will need to move the `"` anyway